civis_jupyter_ext/magics/query.py

```diff
@@ -9,6 +9,22 @@ def magic(line, cell=None):
 
     This magic works both as a cell magic (for table previews) and a
     line magic to query a table and return a DataFrame.
+
+    Examples
+    --------
+    >>> %%civisquery
+    ... SELECT * FROM schema.table;
+
+    >>> %civisquery DATABASE QUERY
+
+    The latter can be useful if you want to use f-strings for variables
+    defined in another cell.
```

Comment:
I think it's worth being explicit that this works for the cell magic too. This statement makes it sound (to me, at least) like it's only for line magic.